src/hooks/users.ts

```diff
@@ -105,6 +108,40 @@ export const useRecentWorldUsers = (): {
   );
 };
 
+type LastSeenAt = {
+  venueName: string;
+  venueUrl: string;
+};
+
+export const useRecentWorldUser = (
+  userId?: string
+): {
+  user?: WithId<User>;
+  lastSeenAt: LastSeenAt;
+} => {
+  const { recentWorldUsers } = useRecentWorldUsers();
+
+  const user = useMemo(() => {
+    return recentWorldUsers.find((worldUser) => worldUser.id === userId);
+  }, [userId, recentWorldUsers]);
+
+  const venueName = getUserCurrentLocation(user) ?? "";
+  const venueUrl = createUrlSafeName(venueName);
+
+  const lastSeenAt: LastSeenAt = {
+    venueName,
+    venueUrl,
+  };
+
+  return { user, lastSeenAt };
+};
+
+export const useRecentUserStatus = (userId?: string): RecentUserStatusType => {
+  const { user } = useRecentWorldUser(userId);
+
+  return getUserDisplayStatus(user);
+};
```

Comment:
This probably doesn't need to be a custom hook, we can just call the `getUserDisplayStatus` within the component/etc that needs it I expect.

Reply:
updated this hook. fixed in 5a3581bc4abfb14880e2e621a12c0994e01cace8